transactions.md

```diff
@@ -129,15 +130,15 @@ vault txout (it is therefore another deposit transaction).
 
 ## emergency_txs
 
-Emergency transactions are used as deterrents against threat. They lock coins to what we
-call an EDV (Emergency Deep Vault): a script chosen by the participants and kept
-obfuscated by the properties of P2WSH, as the emergency transactions are never meant
-to be used.
+Emergency transactions are used as deterrents against threats targetting stakeholders' 
+funds. They lock coins to what we call an EDV (Emergency Deep Vault): a script chosen 
+by the participants and kept obfuscated by the properties of P2WSH, as the emergency 
+transactions are never meant to be used.
```

Comment:
I agree that my "against threat" was not specific enough. However "threats targetting stakeholders' funds" is no better ? All threats are against the stakeholders' funds.

Reply:
Not really. Threats can be against stakeholders themselves, privacy, operational continuity...

Reply:
Right